service/src/test/java/bio/terra/cbas/dependencies/wes/TestCromwellService.java

```diff
@@ -0,0 +1,100 @@
+package bio.terra.cbas.dependencies.wes;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import cromwell.client.model.FailureMessage;
+import java.util.List;
+import org.junit.jupiter.api.Test;
+
+class TestCromwellService {
+
+  @Test
+  void oneFailureNoCausedBy() {
+
+    List<FailureMessage> input = List.of(new FailureMessage().message("Fail message"));
+    String expected = "Fail message";
+
+    String actual = CromwellService.getErrorMessage(input);
+    assertEquals(expected, actual);
+  }
+
+  @Test
+  void listFailMessagesIsNull() {
+    List<FailureMessage> input = List.of(new FailureMessage());
+    String expected = "";
```

Comment:
I was wondering what if we return `null` when there are no errors?

Reply:
I think it may be worth adding a check in `SmartRunsPoller`. When something fails in Cromwell, an `EXECUTOR_ERROR` is returned. I'm assuming we can also get an executor error from various other failures on the  CBAS side as well, so I think checking if we get a null message would omit needing a Cromwell placeholder?

Reply:
I'm assuming that every Cromwell error is a `EXECUTOR_ERROR`, but not every `EXECUTOR_ERROR` is a Cromwell level error. So I'm wondering if we should add a check to see if the message returned is an empty string, then the message in the db should be "null" and the message from the error should be handled elsewhere as it isn't a Cromwell-level error. Thoughts?

Reply:
> I was wondering what if we return `null` when there are no errors?

In this test in particular, it fails when there are no errors, because the function handles a `null` message by returning `""`. It then will call further errors because of the `causedBy` block in a `failures` field. I figured adding a check in the poller is probably the simplest way to handle no errors.